quickstarts/microsoft.recoveryservices/recovery-services-create-vault-with-backup-policies/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vaultName": {
```

Comment:
remove this since you have a defaultValue

Reply:
done. removed param, left empty param file for checks to pass.